gxa/src/main/java/uk/ac/ebi/atlas/home/HomeController.java

```diff
@@ -76,17 +104,34 @@ public String getHome(Model model) {
 
         model.addAllAttributes(latestExperimentsService.fetchLatestExperimentsAttributes());
 
-        model.addAttribute(
-                "speciesList", GSON.toJson(popularSpeciesService.getPopularSpecies(FEATURED_SPECIES)));
-        model.addAttribute(
-                "animalsList", GSON.toJson(popularSpeciesService.getPopularSpecies("animals", FEATURED_SPECIES)));
-        model.addAttribute(
-                "plantsList", GSON.toJson(popularSpeciesService.getPopularSpecies("plants", FEATURED_SPECIES)));
-        model.addAttribute(
-                "fungiList", GSON.toJson(popularSpeciesService.getPopularSpecies("fungi", FEATURED_SPECIES)));
-
         model.addAttribute("speciesPath", ""); // Required by Spring form tag
 
+        List<ExperimentInfo> experimentlist = experimentInfoListService.listPublicExperiments();
+
+        Double numberOfAssays = 0.0;
+        ArrayList<String> numberOfSpecies = new ArrayList<>();
+
+        for (int i = 0; i < experimentlist.size(); i++) {
+            numberOfAssays += experimentlist.get(i).getNumberOfAssays();
+            if (!numberOfSpecies.contains(experimentlist.get(i).getSpecies())) {
+                numberOfSpecies.add(experimentlist.get(i).getSpecies());
+            }
+        }
+
+        model.addAttribute("numberOfStudies", experimentlist.size());
+        model.addAttribute("numberOfAssays", Math.round(numberOfAssays));
+        model.addAttribute("numberOfSpecies", numberOfSpecies.size());
+
+        Map<String, String> atlasInformation = atlasInformationDao.fetchAll();
+        model.addAttribute("info", atlasInformation.toString()
+                .replace("ensemble", "Ensembl")
+                .replace("genomes", "Ensembl Genomes")
+                .replace("paraSite", "WormBase paraSite")
+                .replace("efo", "EFO")
+                .replace("=",":")
```

Comment:
If you find yourself doing any kind of manipulation on a string, ask yourself if there are better approaches for what you are trying to do. It is not necessary to build the sentence "Ensemble x, Esembl Genomes y, etc etc" in the controller. Why can't you use the same clean approach as you did to display the number of studies and assays?